components/Hero/Hero.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { Meta, StoryFn } from "@storybook/react";
+import { Hero, HeroProps } from "components/Hero/Hero";
+
+export default {
+  title: "Hero",
+  component: Hero,
+  args: {
+    title:
+      "Adaptamos y prestamos juguetes para niños y niñas con discapacidades motrices complejas.",
+    subTitle: "El prestamo no tiene costo",
+    image: {
+      src: "https://i.imgur.com/JujA9mV.jpg",
+      alt: "Hero imagen",
+    },
+  },
+  parameters: {
+    nextjs: {
+      appDirectory: true,
+    },
+  },
+} as Meta<typeof Hero>;
+
+const Template: StoryFn<HeroProps> = (args) => <Hero {...args} />;
```

Comment:
Remember that we are using Storybook 7. [Here's how you can write stories](https://storybook.js.org/docs/react/writing-stories/introduction), or use the other stories in our codebase as examples